### DISCUSSION TITLE
How to build curl with only core support ?

### DISCUSSION THREAD

**Discussion by Nemirtingas** [microsoft/vcpkg#20442] [Q&A] [answered]
Hi, I'd like to build curl with only core and winssl support on Windows, but when explicitly using the features I want, it always add the non-http feature, how to disable all default features without modifying the port ?

I though it was possible to disable all features by explicitly using the features.

```
vcpkg.exe install curl[core,winssl] 
Computing installation plan...
The following packages will be built and installed:
    curl[core,non-http,schannel,ssl,sspi,winssl]
```

**Comment by dg0yt** [comment, marked as answer]
`winssl` is just a legacy alias for `schannel`.
Feature `schannel` depends on feature `sspi`.
So these three come together for `winssl`.

But `non-http` and `ssl` shouldn't be there. 

Usual question: Did you run `git pull` and `bootstrap.bat` to ensure the latest vcpkg setup? I cannot reproduce the issue (in a dry run on an x64-linux host):
~~~
$ ./vcpkg install --dry-run curl[core,winssl]:x64-windows
Computing installation plan...
The following packages will be built and installed:
    curl[core,schannel,sspi,winssl]:x64-windows -> 7.79.1
  * zlib[core]:x64-windows -> 1.2.11#12
Additional packages (*) will be modified to complete this operation
~~~

Note that this installation plan prints versions, but yours doesn't.

**Reply by Nemirtingas** [reply]
You're right, it seems this has been fixed in some newer version of vcpkg.
